service/controller/resource/nodes/status.go

```diff
@@ -1,14 +1,18 @@
 package nodes
 
 import (
+	"context"
+
 	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/microerror"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func (r *Resource) GetResourceStatus(customObject providerv1alpha1.AzureConfig, t string) (string, error) {
+	ctx := context.Background()
```

Comment:
We should add context as a parameter to the `GetResourceStatus` func and then propagate it from the caller.

Reply:
Good catch. I missed this when I was getting it to compile.Fixed.